static/js/util/redux.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import type {
+  ActionCreator,
+  AsyncActionHelper,
+  AsyncActionCreator
+} from '../flow/reduxTypes';
+import type { Dispatch } from 'redux';
+
+export function createActionHelper(dispatch: Dispatch, actionCreator: Function): (...args: any) => void {
+  return (...args) => dispatch(actionCreator(...args));
+}
+
+export type ActionHelpers = Array<{[k: string]: (...args: any) => void}>;
+export function createSimpleActionHelpers(dispatch: Dispatch, actionManifest: ActionCreator[]): ActionHelpers {
+  return actionManifest.map(actionCreator => (
+    { [actionCreator.name]: createActionHelper(dispatch, actionCreator) }
+  ));
+}
+
+export type AsyncActionHelpers = Array<{[k: string]: AsyncActionHelper}>;
+export function createAsyncActionHelpers(dispatch: Dispatch, actionManifest: AsyncActionCreator[]): AsyncActionHelpers {
+  return actionManifest.map(actionCreator => (
+    { [actionCreator.name]: createActionHelper(dispatch, actionCreator) }
+  ));
+}
```

Comment:
can you leave a brief comment for possibly-confused devs (read: me) looking at this in the future? aside from that, this looks good